docs/courses/README.md

```diff
@@ -1,28 +1,104 @@
+
+
+<a id="exploring-courses"></a>
+### Exploring the courses
+
+To have a better understanding of how courses work, you can explore how the course files are organized on this repository.
+
+You will find courses in the `courses` directory of [LibreLingo's repository](https://github.com/kantord/LibreLingo/tree/main/courses). The course directory name is usually structured in `<destination language>-from-<target audience's language>`. e.g. `french-from-english` teaches French to English speakers. In each course you'll find modules. And in each module you'll find skills, which are saved as individual .yaml files.
+
+Now let's look into how they look on the frontend, to the user.
+
+Go to [LibreLingo web app](https://librelingo.app/). Courses are listed there. e.g. You see a "START LEARNING SPANISH" button. (**NOTE:** Depending on the state and completeness of the courses, all of them might not be shown there on the website.)
+
+When you enter a course page, for example the [Spanish course](https://librelingo.app/course/spanish-from-english/), you'll see headings like Basics, Introduction, Activities. These are modules. Each module contains a set of skills. Modules are merely there to serve the purpose of organizing skills. Think of them as groups of units or chapters.
+
+"Skills" are units of course content focused on a particular topic. Each skill is a set of new knowledge, such as new words or phrases.  Skills can focus on vocabulary and follow a theme: the "Animal" skill has words and phrases related to animals, the "Food" skill is all about food. Skills can also focus on teaching grammar.
+
+Inside a skill YAML file there are some keys that you might want to learn of, which is exaplained in detail [here](course.md).
+
+<a id="things-new-contributors"></a>
+### Things you can do as a new contributor
+
+If you are new in contributing to course data, these are the things you can do to get a proper understanding of the courses and have a good time with the project:
+
+- Go to [LibreLingo's Spanish course](https://librelingo.app/course/spanish-from-english) and try out some of the skills. Just get an idea on how the questions are structured and the type of answers made available.
+- Clone this repository or [download it](https://github.com/kantord/LibreLingo/archive/main.zip) and study the `courses` directory and how everything's laid out. Be sure to have a good text editor, such as [Notepad++](https://notepad-plus-plus.org/), [Geany](https://geany.org/) in hand to view or edit files.
+- Try to learn how data is organized within .yaml files and directories. [Course](course.md), [Module](module.md) and [Skill](skill.md) documentation may help you on this.
+- Try to learn how YAML works. (see [here](https://en.wikipedia.org/wiki/YAML#Syntax) or [here](https://blog.stackpath.com/yaml/) or [here](https://www.cloudbees.com/blog/yaml-tutorial-everything-you-need-get-started/))
+- Determine some information about your language.
```

Comment:
```suggestion
- Find some information about the language you're working with
```

Although maybe this is only relevant for people who are creating a new course from scratch

Reply:
This is also applicable for translations because course editors will have to change the language code, name etc. according to their language.

Reply:
"Find" seems actually applicable for the BCP 47 point. I think "determine" is a better word for other 2 points. Although I think "determine" if fine for all 3 of them. So I suggest:

```suggestion
- Determine some information about the language you're working with.
```